src/test/java/net/masterthought/cucumber/generators/integrations/FailuresOverviewPageIntegrationTest.java

```diff
@@ -69,21 +69,4 @@ public void generatePage_onJsonWithoutFailedSteps_generatesProperMessage() {
         SummaryAssertion summary = document.getReport();
         assertThat(summary.getEmptyReportMessage()).isEqualTo("You have no failed scenarios in your Cucumber report");
     }
```

Comment:
They were testing methods that escaped data. These methods were no longer used and thus removed, leaving nothing to be tested.